src/Microsoft.AspNetCore.SignalR.Core/Hub.cs

```diff
@@ -6,13 +6,19 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A base class for a SignalR hub.
+    /// </summary>
     public abstract class Hub : IDisposable
     {
         private bool _disposed;
         private IHubCallerClients _clients;
         private HubCallerContext _context;
         private IGroupManager _groups;
 
+        /// <summary>
+        /// Gets or sets the clients caller.
```

Comment:
`Gets or sets an object that can be used to invoke methods on the clients connected to this hub`